app/services/users/auth-service.ts

```diff
@@ -0,0 +1,44 @@
+import {Component} from 'angular2/angular2';
+import {Http, HTTP_PROVIDERS, Headers, Response} from 'angular2/http';
+
+@Component({
+    providers: [Http, HTTP_PROVIDERS]
+})
+export class AuthService {
+    public token;
+    public isConnected;
+    private clientId = 'Bziiu7E0wFtNre6TLdSnEqIe73prHbjJsPx3p5rL';
+    private clientSecret = '8QCDHhiwwAEA5RIwuta8SpprcoJPmpfIwhkPNZil8wJqeLuAh6BKsvWVxNkFvg5YgO0aFrsYngY7YK77iil9f4jRFNSBX11XCe0iGTUd0o9HKCvj72twl0qMT6hlJp3l';
```

Comment:
On pourrait faire un fichier secrets.js qui serait importé par les parties de l'app qui en ont besoin, et qui serait hors git. Mais de toutes façons c'est dans le client, donc ça sera public in fine... C'est surtout ça que je trouve bizarre